src/DataSource/Http.hs

```diff
@@ -66,6 +68,7 @@ emptyZendeskLayer = ZendeskLayer
     { zlGetTicketInfo           = \_     -> error "Not implemented zlGetTicketInfo!"
     , zlListRequestedTickets    = \_     -> error "Not implemented zlListRequestedTickets!"
     , zlListAssignedTickets     = \_     -> error "Not implemented zlListAssignedTickets!"
+    , zlListAdminAgents         =           error "Not implemented zlListAgents" -- This returns error during testing
```

Comment:
Any test that uses ZendeskLayer fails because of this. Do you know any workaround for this?

Reply:
Ah, yes. It's something I lost a ton of time on - for now, you can just do `pure []` and be done with it, otherwise the `error` get's evaluated immediately - there is no parameter for it, see?

Reply:
Done! But I had to add Monad constraint to the emptyZendeskLayer (could be applicative)